### DISCUSSION TITLE
Show in File Manager issues

### DISCUSSION THREAD

**Discussion by rhodesilent** [atom/atom#22854] [Q&A]
(First time posting here: Apologies if this is not the correct place to ask this)

I am using Atom version 1.20.2 on Ubuntu Desktop 18.04. I want to be able to open location of a file by right clicking on it in the project pane: 

![image](https://user-images.githubusercontent.com/60770864/129445147-9826f05c-4bf1-4d5a-a653-a608425c61ab.png)

But I see this instead:

![image](https://user-images.githubusercontent.com/60770864/129445136-56fc20b7-9012-4a35-95ee-b4c23abed3bd.png)

Any ideas how this could be fixed?

**Comment by idleberg** [comment]
No solution, just an alternative:

https://github.com/idleberg/atom-browse
